packages/@okta/vuepress-site/docs/guides/ie-limitations/index.md

```diff
@@ -47,6 +59,26 @@ The following Event Types are available only in the Identity Engine and can't be
 
 ***
 
+#### Reset Factor API - email enrollment
+
+**What Changed:** A user’s verified `primaryEmail` is considered an email (Authenticator) enrollment for the user. The use of the Classic Engine Reset Factor API for resetting a user’s email enrollment is discouraged and considered moot, because email is an auto-enrolling Authenticator in Identity Engine. A user’s verified `primaryEmail` is always usable as long as the Email Authenticator is set to `ACTIVE`, and the user can use it for **recovery only** or for both **authentication and recovery**, depending on the Email Authenticator settings.
+
+***
+
+#### Reset Factor API - recovery question
```

Comment:
This LGTM on a high level. 
Just a few thoughts to ponder based on the [jira discussion(s)](https://oktainc.atlassian.net/browse/OKTA-434579?focusedCommentId=2158471) : 

1. Should we reason out this limitation is because the classic GET factors API only returns one factor enrollment at a time (factor cardinality = 1 in classic)
2. Should we care to mention the `ALLOW_USING_SQ_ENROLLMENT_FROM_OIE_IN_V1` FF, which makes the recovery question also usable for AuthN from the classic factors/authn API? 
3. Should we mention this is specific to upgrade org scenarios only - i.e. only classic orgs which have both Security Question For MFA and Forgot Password Question enabled and there have users that have 2 SQ enrollments (MFA question and Recovery Question).

I will let Bhagya make the final call on this, based on this data/background.

Reply:
I think we should enumerate points 1 and 4 to make customers aware.  
Let us hold on 2 until we can GA the flag